### ISSUE TITLE
run syslog server for all non-kubenetes & external logs

### ISSUE THREAD

**Issue by billimek** [billimek/k8s-templates#88]

**Comment by billimek** [comment]
Experimented with fluentd as well as logstash to get a syslog server running but was not successful.  Will explore the configoverride feature of fluent-bit again as soon as [this PR](https://github.com/helm/charts/pull/8960) is merged.

**Comment by billimek** [comment]
Waiting on [charts PR#8960](https://github.com/helm/charts/pull/8960) for this.

**Comment by runningman84** [comment]
You might want to try this as a lightweight alternative to the resource hungry elk stack:
https://grafana.com/loki

**Comment by billimek** [comment]
> You might want to try this as a lightweight alternative to the resource hungry elk stack:
> https://grafana.com/loki

Yeah - I signed-up when it was announced at KubeCon, but haven't installed the forwarding agent yet.  Can't wait to take a look at this!
